woocommerce/woocommerce
# New empty state for variations

### All Submissions:






### Changes proposed in this Pull Request:



This PR adds another empty state when the attributes with the `Used for variations` box are added, but the variations are not generated yet. There's an image and copy that reads `No variations yet. Generate them from all added attributes or add a new variation manually.` There's also an arrow that points to the new "Generate variations" button.

Closes #37147.



### How to test the changes in this Pull Request:




1. Go to Products > Add New > Product data
2. Select `Variable product`.
3. Press `Attributes` and add (or create) a new Attribute.
4. Press `Variations` and verify that the new screen is visible.
5. Press `Generate variations` and verify that the variations are created as expected.
6. The old dropdown menu and the `Go` button are still visible.
7. Remove the created variations and verify that the new screen is visible again.
8. Press `Add manually` and verify that the form to create a variation is visible.




### Acceptance criteria
- [ ] Users will see this screen when the product type is variable and there’s at least one attribute added with the `Used for variations` box checked.
- [ ] Add a `Generate variations` button that generates the variations from all attributes with the `Used for variations` box checked.
- [ ]  Add an `Add manually` button that adds a new variation form (the same as it does now).
- [ ] Hide the dropdown menu.
- [ ] Show the text: No variations yet. Generate them from all added attributes or add a new variation manually.


### Other information:




### FOR PR REVIEWER ONLY:
